static/app/utils/performance/histogram/histogramQuery.tsx

```diff
@@ -81,12 +82,13 @@ function HistogramQuery(props: Props) {
     );
   }
 
+  const {children: _, ...propsWithoutChildren} = props;
   return (
     <GenericDiscoverQuery<Histograms, HistogramProps>
       route="events-histogram"
       getRequestPayload={getHistogramRequestPayload}
       didFetch={didFetch}
-      {...omit(props, 'children')}
+      {...propsWithoutChildren}
```

Comment:
On line 94, props.children can be changed into children since we destructure it now.